docker/Dockerfile

```diff
@@ -1,40 +1,34 @@
-FROM golang:alpine as gobuild
+### Webapp build
+FROM node:16.3.0 as nodebuild
 
-RUN apk update && \
-    apk add ca-certificates gcc git make musl-dev && \
-    git clone https://github.com/mattermost/focalboard && \
-    cd focalboard && \
-    make server-linux
+WORKDIR /webapp
+ADD webapp/ /webapp
 
-FROM node:alpine as nodebuild
+RUN npm install --no-optional && \
+    npm run pack
```

Comment:
I think we should avoid directly running npm commands and instead use `Makefile` as the entry point and interface for all things DevOps.

For example -

```
RUN npm install --no-optional &&\
cd .. && make webapp
```

This allows us to encapsulate the compilation and build process and avoid frequent change to DevOps scripts. This should be discussed in the Focalboard channel as well.

Reply:
I think both options are ok, actually I almost prefer to not have the Makefile make the frontend and the backend work independently, but I understand that is really handy for others. I think the webapp should be compilable without the makefile with one right command.

Reply:
i will keep as-is for now